model/adapter.py

```diff
@@ -393,6 +393,8 @@ def knockout_gene(self, feature):
         :param gene_name: gene name
         :return:
         """
+        if self.model.reactions.has_id(reaction_id):
```

Comment:
the speed is not fixed, still need to figure out. That is the fix for adding the existing reaction